app/assets/javascripts/file-input-handler.js

```diff
@@ -12,17 +12,21 @@ function toggleSubmitButton(enabled) {
     }
 }
 
-function onFileSelected(id, targetURL, csrf) {
+function onFileSelected(id, targetURL) {
     return function(event) {
         const files = event.target.files;
 
+        const csrfToken = document.querySelector('input[name="csrfToken"]').value;
+
+//        document.write (csrfToken); // Please only use this for manual testing purposes. Uncomment this to test the csrf token or use console and check document.querySelector('input[name="csrfToken"]').value;
```

Comment:
Please delete this once the page as been tested. A way to display the csrf token that has been added to the view is with `document.write`.

This view below doesn't currently have a form with csrf token. This has been added so when you get to this view, inspect and grab the csrf token. The csrf token when the file is chosen should match when `document.write (csrfToken)` is activated. 

You can also just use the console to check `document.querySelector('input[name="csrfToken"]').value;` which should be the same value.

<img width="1556" alt="Screenshot 2024-05-21 at 15 45 25" src="https://github.com/hmrc/binding-tariff-trader-frontend/assets/63714619/0310aa66-d998-4bed-8ebc-d1713c762534">